libs/ngrx-toolkit/src/lib/with-undo-redo.ts

```diff
@@ -143,7 +144,7 @@ export function withUndoRedo<Input extends EmptyFeatureResult>(
     })),
     withHooks({
       onInit(store) {
-        effect(() => {
+        watchState(store, () => {
```

Comment:
This change is great and allows us to make all tests in `with-und-redo.spec.ts` synchronous.

Can we omit all `fakeAsync`- and `tick`-usages there?